pages/common/csvquote.md

```diff
@@ -0,0 +1,20 @@
+# csvquote
+
+> Enables common unix utlities like cut, awk, wc, head to work correctly with csv data containing quoted delimiters and newlines
+> More information: <https://github.com/dbro/csvquote>.
+
+- Replace quoted special characters in normal csv file:
+
+`csvquote {{data.csv}}`
+
+- Replace quoted special characters in normal tsv file:
+
+`csvquote -t {{data.csv}}`
+
+- Restore quoted special characters from stdin:
+
+`csvquote -u`
+
+- Most common use case:
```

Comment:
Does this really explain what the example does?

Reply:
I agree that it is confusing and could be clear. Please mention what the example is doing rather than just the fact it is the most common use case.

Reply:
@tianshuo you missed this comment

Reply:
> @tianshuo you missed this comment

I have added a new commit